### ISSUE TITLE
Is satiator-types.h really GPL?

### ISSUE THREAD

**Issue by slinga-homebrew** [satiator/satiator-menu#1]
EOM

**Comment by abrasive** [comment]
I think the Satiator "library" that you can reuse in other things should be public domain. I'll move it to its own repo when I get a chance.

**Comment by slinga-homebrew** [comment]
Problem is the Satiator library uses Iapetus which is GPL. I don't mind making Save Game Copier GPL (which it is) but a more permissive license might make it easier for Jo Engine and Yaul to add support. Assuming of course they are interested.

**Comment by abrasive** [comment]
The only thing it uses is `cd_stop_drive`, right? Easy enough to reimplement.

**Comment by abrasive** [comment]
To be clear, I mean reimplement in `satiator.c` - so it can be standalone.

**Comment by slinga-homebrew** [comment]
Yeah cd_stop_drive() is the only function as far as I can tell. And a bunch of register and IRQ definitions from the header file.
